docs/pages/guides/testing-rsc.mdx

```diff
@@ -48,7 +48,7 @@ Then, you can add a script such as `test:rsc` to your **package.json**:
 
 Tests should be written in a **\_\_rsc_tests\_\_** directory to prevent Jest from running your client tests on the server.
 
-```js __rsc_tests__/my-component.test.ts
+```tsx __rsc_tests__/my-component.test.ts
```

Comment:
````suggestion
```ts __rsc_tests__/my-component.test.ts
````

Because file format is `.ts`.

Reply:
@amandeepmittal The first argument is the language we provide for Prism, when setting it to `ts` the JSX elements are not highlighted correctly, so `<LinearGradient ...>` code in this snippet.